doc/self-host.md

```diff
@@ -0,0 +1,31 @@
+# Hosting a Gateway
+
+## build
+```shell
+npm run build
+```
+
+## start
+The server needs at least the following environment variables:
+
+| variable                  | example                                   | comments            |
+|---------------------------|-------------------------------------------|---------------------|
+| M_LD_GATEWAY_AUTH__KEY    | `rootacc.keyid:0gcsqgsib3dqebaquaa4gna`   | Root API access key |
+| M_LD_GATEWAY_KEY__PUBLIC  | `MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKB...` | Root public key     |
+| M_LD_GATEWAY_KEY__PRIVATE | `MIIC3TBXBgkqhkiG9w0BBQ0wSjApBgkqhkiG...`  | Root private key    |
+| M_LD_GATEWAY_GATEWAY      | `my-imac.local`                           | External hostname   |
+| M_LD_GATEWAY_DATA_PATH    | `./local/edge/gw/data`                    | Path for data       |
```

Comment:
**suggestion:** Let's clarify what should be here. Is this a directory? A file?

Reply:
Agreed, will do.